core/src/main/java/com/github/mizool/core/validation/CheckCountryCode.java

```diff
@@ -39,7 +39,7 @@ public void initialize(CountryCode countryCode)
     @Override
     public boolean isValid(String validationObject, ConstraintValidatorContext constraintValidatorContext)
     {
-        return isNullButOptional(validationObject) || isValidValue(validationObject);
+        return isNullButOptional(validationObject) || (validationObject != null && isValidValue(validationObject));
```

Comment:
Create a package private utility class `ConstraintValidators` looking something like this:
```java
public static <T> boolean isValid(T validationObject, boolean mandatory, Predicate<T> isValidValue)
{
    return isNullButOptional(validationObject, mandatory) || isNotNullAndValid(validationObject, isValidValue);
}

private static <T> boolean isNullButOptional(T validationObject, boolean mandatory)
{
    return validationObject == null && !mandatory;
}

private static <T> boolean isNotNullAndValid(T validationObject, Predicate<T> isValidValue)
{
    return validationObject != null && isValidValue.test(validationObject);
}
```
Replace this expression inside all validators with
```java
return ConstraintValidators.isValid(validationObject, mandatory, this::isValidValue);
```